jkube-kit/common/src/test/java/org/eclipse/jkube/kit/common/ConfigsAsBooleanTest.java

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright (c) 2019 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at:
+ *
+ *     https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.jkube.kit.common;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+import java.util.Arrays;
+import java.util.Collection;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@RunWith(Parameterized.class)
+public class ConfigsAsBooleanTest {
+
+  @Parameterized.Parameters(name = "{index}: asBoolean({1})={2}")
```

Comment:
You're not adding description parameter here. 
```suggestion
  @Parameterized.Parameters(name = "{index} {0} : asBoolean({1})={2}")
```